.github/workflows/deploy.yml

```diff
@@ -64,15 +64,15 @@ jobs:
                   DOPPLER_TOKEN: ${{ secrets.DOPPLER_PROD_AWS_LAMBDAS_SECRET }}
               run: |
                   aws lambda update-function-configuration --function-name getSessionIdsByQuery \
-                    --environment "$(doppler secrets download --no-file | jq '{Variables: .}')"
+                    --environment "$(doppler secrets download --no-file | jq '. + {REACT_APP_COMMIT_SHA: ${{github.sha}} }' | jq '{Variables: .}'"
```

Comment:
are the `$(` not terminated now?

Reply:
Can you propose an alternative? I don't know how to actually test a github action to verify the behavior.

Reply:
@et You can test locally by running `echo "$(doppler secrets download --no-file | jq '. + {REACT_APP_COMMIT_SHA: ${{github.sha}} }' | jq '{Variables: .}'"` - should error

i think you need to keep the `env:` values you had, and then use the `$REACT_APP_COMMIT_SHA` value as so:

```bash
doppler secrets download --no-file | jq ". + {REACT_APP_COMMIT_SHA: $REACT_APP_COMMIT_SHA }" | jq '{Variables: .}'
```

so it would be
```bash
aws lambda update-function-configuration --function-name getSessionIdsByQuery --environment "doppler secrets download --no-file | jq \". + {REACT_APP_COMMIT_SHA: $REACT_APP_COMMIT_SHA }\" | jq '{Variables: .}'"
```

<img width="967" alt="Screenshot 2023-08-01 at 10 05 46 AM" src="https://github.com/highlight/highlight/assets/1351531/bd2ea9a0-dc72-48a8-bb1c-c2e2b849dc15">

to test, you can also make this yml file run in your branch to make sure it will correctly run once merged to main. you can add the following in a temp commit, push that, then push a revert 
edit the following at the top
```yaml
on:
    ... what's there now ...
    pull_request:
        types: [opened, synchronize]
```